api/db/changelog/db.changelog-183-new-user-recent-resource-table.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="neha" id="changelog-183-new-user-recent-resource-table">
+    <createTable tableName="user_recent_resources">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="workspace_id" type="bigint">
+        <constraints nullable="false"/>
+      </column>
+      <column name="resource_type" type="tinyint">
```

Comment:
We have a new way of storing enums in the DB which seems to be working well so far.  This might be another good place to try that.  What do you think?

I'm referring to how we specify `DbAccessModule`:
```
<column name="name" type="ENUM('ERA_COMMONS', 'TWO_FACTOR_AUTH', 'RAS_LOGIN_GOV', 'RT_COMPLIANCE_TRAINING', 'DATA_USER_CODE_OF_CONDUCT', 'PROFILE_CONFIRMATION', 'PUBLICATION_CONFIRMATION')">
```